pkg/local_object_storage/metabase/inhume.go

```diff
@@ -164,10 +186,12 @@ func (db *DB) Inhume(prm InhumePrm) (res InhumeRes, err error) {
 			obj, err := db.get(tx, prm.target[i], buf, false, true, currEpoch)
 			targetKey := addressKey(prm.target[i], buf)
 			if err == nil {
+				containerID, _ := obj.ContainerID()
```

Comment:
Why have you decided to have it outside `if`?

Reply:
It is inside the `if` now.

Reply:
I mean, it always was inside the `if`. Do you mean something else?